cdf-pentaho5/resource/settings.xml

```diff
@@ -23,6 +23,13 @@
 
         accepted values are: true | false
      -->
-    <legacy-dashboard-context>false</legacy-dashboard-context>    
+    <legacy-dashboard-context>false</legacy-dashboard-context>
+
+    <!--
```

Comment:
Document this flag - do not push changes to settings.xml.

Make sure code works without the flag in settings.xml and that it defaults to false.